src/components/POLine/POLineDetails/FieldOrderFormat.js

```diff
@@ -51,6 +52,12 @@ class FieldOrderFormat extends Component {
         dispatch(change('eresource.activationDue', vendor.expected_activation_interval));
       }
     }
+
+    if (value === OTHER) {
+      dispatch(change('physical.createInventory', createInventorySetting.other));
+    } else {
+      dispatch(change('physical.createInventory', createInventorySetting.physical));
+    }
   }
```

Comment:
It seems for me this method is becoming too big and complex. Colleagues, what do you think about creating separate method for processing changing order format and moving it to /Utils for instance?

Reply:
I think it's better to keep in one place till we have all the requirements. After that we can cover it with tests properly and refactor